docs/sql-data-sources-avro.md

```diff
@@ -234,7 +234,8 @@ Data source options of Avro can be set via:
           When reading Avro, this option can be set to an evolved schema, which is compatible but different with
```

Comment:
```suggestion
          When reading Avro files or calling function <code>from_avro</code>, this option can be set to an evolved schema, which is compatible but different with
```

Reply:
In line 247 we mention "read, write and function <code>from_avro</code>". Let's mention the function call here too.